server.js

```diff
@@ -12,8 +17,94 @@ app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-app.use(express.static(path.resolve(__dirname, './build')));
+app.use(express.static(path.resolve(__dirname, './public')));
 app.use('/api/v1', router);
-app.get('*', (req, res) => res.sendFile(path.resolve(__dirname, './build', 'index.html')));
+app.get('/*', (req, res) => res.sendFile(path.resolve(__dirname, './public', 'index.html')));
+
+
+passport.serializeUser(function(user, done) {
+  done(null, user);
+});
+
+passport.deserializeUser(function(obj, done) {
+  done(null, obj);
+});
+
+
+passport.use(new GitHubStrategy({
+    clientID: keys.CLIENT_ID,
+    clientSecret: keys.CLIENT_SECRET,
+    callbackURL: "http://localhost:3000/auth/github/callback"
```

Comment:
I think this might be causing you some issues because this is actually never being hit.